app_dart/lib/src/service/github_service.dart

```diff
@@ -284,4 +284,45 @@ class GithubService {
     final Random rnd = Random();
     return String.fromCharCodes(Iterable<int>.generate(10, (_) => chars.codeUnitAt(rnd.nextInt(chars.length))));
   }
+
+  /// Get content of [filePath] from GitHub CDN.
+  Future<String> githubFileContent(
```

Comment:
nit: Since this is in `GithubService`, you can simplify the method name to something like `fileContent`